src/main/java/com/zaiats/catcher/model/CandidateModel.java

```diff
@@ -2,14 +2,27 @@
 
 import com.zaiats.catcher.repository.entity.Candidate;
 
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.Size;
 import java.util.Objects;
 
 public class CandidateModel {
 
     private Long id;
+
+    @NotBlank
+    @Size(max = 64)
     private String firstName;
```

Comment:
you need to add the value min (2-3 letters)

Reply:
It should have at least 1 character according to the pair of those validation annotations.

Reply:
Both annotations in pair means 1-64 characters.